src/ml/data/dwh/setup_dwh.py

```
@@ -53,226 +63,251 @@ def setup_logging(level: str = "INFO") -> None:
-        format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
         handlers=[
             logging.StreamHandler(sys.stdout),
-            logging.FileHandler('dwh_setup.log')
-        ]
+            logging.FileHandler("dwh_setup.log"),
+        ],
     )
 
 
 def main():
```

Pylint (3.3.7)

[refactor] Too many local variables (23/15)

(R0914)